test/types.spec.ts

```diff
@@ -0,0 +1,45 @@
+import assert from "assert";
+import bytes from "bytes-iec";
+
+/*
+this test is intended more to verify type definitions than functionality
+*/
+
+describe("Entrypoints", () => {
+	it("is callable (`format`)", () => {
```

Comment:
the project is in 2-spaced formatting.  Not my choice :P

Reply:
Yeah I've never been a fan of how that suddenly became a thing everywhere... Same with how single quotes are suddenly the hot new string delimiter

Reply:
Do you mind if I resolve this with a .editorconfig to avoid this discussion in future work?